oozie/src/main/java/org/apache/falcon/oozie/OozieOrchestrationWorkflowBuilder.java

```diff
@@ -219,6 +230,25 @@ protected void decorateWorkflow(WORKFLOWAPP wf, String name, String startAction)
         wf.getDecisionOrForkOrJoin().add(kill);
     }
 
+    protected void addPostProcessing(WORKFLOWAPP workflow, ACTION action) throws FalconException{
```

Comment:
This is missing addHDFSServersConfig() method, which was added for replication workflows. But, I don't think it is needed for post-processing (should be sufficient for replication). Can you check @sowmyaramesh or @vramachan to confirm